### DISCUSSION TITLE
How to override tailwind.config.js from shared package monorepo

### DISCUSSION THREAD

**Discussion by danilrafiqi** [ben-rogerson/twin.macro#804] [Q&A] [answered]
I made a monorepo using yarn, with the folder structure below.

<img width="293" alt="Screenshot 2023-05-26 at 11 19 07" src="https://github.com/ben-rogerson/twin.macro/assets/22560475/250ae4f4-7bd2-424c-8a5f-8d603ad00152">

shared-component is a component library that I created using twin.macro, I want the default style in the shared-component to be overridden by react-template

**Comment by ben-rogerson** [comment, marked as answer]
Hey there

Have you taken a look at the tailwindcss presets? Could solve the problem:
https://tailwindcss.com/docs/presets
